docs/ibc/light-clients/client-state.md

```diff
@@ -28,6 +28,8 @@ implementer is in charge of determining which checks are required. See the [Tend
 - A `Frozen` status indicates that misbehaviour was detected in the counterparty chain and the client is not allowed to be used.
 - An `Expired` status indicates that a client is not allowed to be used because it was not updated for longer than the trusting period.
 - An `Unknown` status indicates that there was an error in determining the status of a client.
+- An `Unauthorized` status indicates that usage of the client is not allowed.
+  - > Note: Removing a client type from the [`AllowedClients`](https://github.com/cosmos/ibc-go/blob/v6.0.0/modules/core/02-client/types/client.pb.go#L345) array will prevent all usage of that client type.
```

Comment:
Added a note in the client-state.md doc. 

cc @colin-axner

Reply:
Hmm, the only issue with the doc here is that this is for the `clientState.Status` message. Light client developers don't need to use the `Unauthorized` status

Reply:
That is a very good point! Let me see if I can find a better place for this.